app/forms/booking_request_form.rb

```diff
@@ -6,6 +6,24 @@ class BookingRequestForm
                 :memorable_word, :appointment_type, :accessibility_requirements,
                 :opt_in, :dc_pot
 
+  with_options if: :step_one? do |step_one|
+    step_one.validates :primary_slot, presence: true
+    step_one.validates :secondary_slot, presence: true
+    step_one.validates :tertiary_slot, presence: true
+  end
+
+  with_options if: :step_two? do |step_two|
+    step_two.validates :first_name, presence: true
+    step_two.validates :last_name, presence: true
+    step_two.validates :email, presence: true, format: { with: /.+@.+\..+/ }
+    step_two.validates :telephone_number, presence: true
+    step_two.validates :memorable_word, presence: true
+    step_two.validates :appointment_type, inclusion: { in: %w(50-54 55-plus) }
```

Comment:
I'm going to change the flow so when this or the `dc_pot` validation is triggered we redirect to the eligibility screen, rather than presenting them as validation errors.